help/scim.md

```diff
@@ -88,6 +88,26 @@ Zulip's SCIM integration has the following limitations:
        Zulip account to be updated accordingly.
      * Unassigning a user from the app will deactivate their Zulip account.
 
+1. Additionally, Zulip now supports synchronization of [user groups](/help/user-groups)
```

Comment:
Better to drop "now" here -- documenting what is works as better writing, since then we don't need to update the text in a year once it's no longer new.